### ISSUE TITLE
chartgerink: Some edits to your case study

### ISSUE THREAD

**Issue by fatmai** [BIDS/repro-case-studies#135]
Hi @chartgerink, we got further revisions some points came up from one of the reviewers:
- [ ] The Workflow is very general; it doesn't tell much about the specific project. It would be more interesting and engaging if you included specific details of the research - e.g. what does the data look like? What kind of analyses did you run? 
  You mention these in the first paragraph when you introduce the problem but afterwards only describe the workflow in a high level. We would like to see a little more specifics.
- [ ] It would be helpful if you could write in past tense instead of future/present throughout the description of your project. The Workflow section is sometimes hard to follow because of this.
- [ ] Please describe the acronym OSF the first time you use it in the case study.

**Comment by chartgerink** [comment]
Great, thanks! Will get these back to you on Monday at the latest if that's OK. Just a PR on the submissions branch again, I figure?

Edit: the third point is already in there, AFAIK.

**Comment by chartgerink** [comment]
Just created the PR. I felt somewhat uncomfortable about writing in past tense, because I am still not done with the project :-) Let me know if this works for you or you'd like to see some additional edits.

**Comment by fatmai** [comment]
thanks, @chartgerink. np with the past tense. its ok this way. PR #137 looks good, merged. closing the issue.
